libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -611,19 +617,36 @@ bool AP_Follow::handle_global_position_int_message(const mavlink_message_t &msg)
         return false;
     }
 
+    // apply jitter-corrected timestamp to this update
+    uint32_t location_update_ms = _jitter.correct_offboard_timestamp_msec(packet.time_boot_ms, AP_HAL::millis());
+    if (location_update_ms < _last_location_update_ms) {
```

Comment:
this will break with wrap of uint32_t ms, needs to be done via a 32 bit subtraction if you want this at all, I'm not sure what you're trying to achieve

Reply:
needs to use 64 bit timestamps

Reply:
is this right?

Reply:
As discussed, dropping this from this PR so it can be handled properly.